packages/flutter_tools/pubspec.yaml

```diff
@@ -38,7 +38,6 @@ dependencies:
   yaml: 2.1.16
   flutter_goldens_client:
     path: ../flutter_goldens_client
-  protobuf: 0.13.15
```

Comment:
Does this also need to be removed from the BUILD.gn?

Reply:
We technically still depend on this transitively...I'm not sure. I'd rather leave it in to be safe